app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -0,0 +1,46 @@
+class OpenidConnectUserInfoPresenter
+  include Rails.application.routes.url_helpers
+
+  attr_reader :identity
+
+  def initialize(identity)
+    @identity = identity
+  end
+
+  def user_info
+    email = identity.user.email
+
+    {
+      sub: identity.uuid,
+      iss: root_url,
+      email: email,
+      email_verified: email
+    }.merge(loa3_attributes)
+  end
+
+  private
+
+  # TODO: only return loa1 attributes for loa1 sessions
+  def loa3_attributes
+    {
+      given_name: loa3_data.first_name,
```

Comment:
re:
> Is it possible for the OpenId request to specify which attributes are asserted? Likewise, should we be consulting the ServiceProvider definition for which attributes to assert?
  
Yes, that's the `scope` and `acr_values` attribute of the original request. For this working prototype, I did not implement either of those, hence the TODOs I have written, that will be my immediate next step.